### PR TITLE
Implemented screenshot feature

### PR DESCRIPTION
**Pull request by erdemyerebasmaz**
This PR introduces takeScreenshot method to webview plugin that takes the visible view port's screenshot and returns it's byte data as result.

### CODE REVIEW

File: android/src/main/java/com/flutter_webview_plugin/WebviewManager.java [hunk 1]
```diff
@@ -188,6 +190,21 @@ public void onReceiveValue(Boolean aBoolean) {
         }
     }
 
+    void takeScreenshot(MethodCall call, MethodChannel.Result result){
+        if (webView != null) {
+            webView.enableSlowWholeDocumentDraw();
+            webView.setDrawingCacheEnabled(true);
+            Bitmap bitmap = Bitmap.createBitmap(webView.getDrawingCache());
```
Comment (roeierez) [inline android/src/main/java/com/flutter_webview_plugin/WebviewManager.java]: It seems that getDrawingCache is deprecated (https://developer.android.com/reference/android/view/View.html#getDrawingCache())
Look at the deprecation comment at this link, they suggest two alternatives.

**Review by roeierez — CHANGES_REQUESTED** [review]

File: android/src/main/java/com/flutter_webview_plugin/WebviewManager.java (hunk 1 above)
Comment (erdemyerebasmaz) [reply android/src/main/java/com/flutter_webview_plugin/WebviewManager.java]: `
For software-rendered snapshots of a small part of the View hierarchy or individual Views it is recommended to create a Canvas from either a Bitmap or Picture and call draw(android.graphics.Canvas) on the View. However these software-rendered usages are discouraged and have compatibility issues with hardware-only rendering features such as Config.HARDWARE bitmaps, real-time shadows, and outline clipping. `

I initially used this method but there were render bugs which caused 30% from top of the image to be a blank area.

Comment (erdemyerebasmaz) [reply android/src/main/java/com/flutter_webview_plugin/WebviewManager.java]: @roeierez Changed the deprecated method: https://github.com/breez/flutter_webview_plugin/pull/2/commits/98e8f8434b9fab07aec60db62ab4a62f9809a732

File: android/src/main/java/com/flutter_webview_plugin/BrowserClient.java [hunk 2]
```diff
@@ -37,12 +37,45 @@ public void onPageFinished(WebView view, String url) {
...
-                    "window.addEventListener('message', window.breezReceiveMessage, false);" +
+        String listenPostMessagesScript =
+                "if (!window.breezReceiveMessage) {" +
+                "   window.breezReceiveMessage = function(event) {Android.getPostMessage(event.data);};" +
+                "   window.addEventListener('message', window.breezReceiveMessage, false);" +
                 "}";
-        view.evaluateJavascript(script, null);
+        view.evaluateJavascript(listenPostMessagesScript, null);
 
+        String includePromisePolyfill = "var promisePolyfill = document.createElement(\"script\");\n" +
```
Comment (roeierez) [inline android/src/main/java/com/flutter_webview_plugin/BrowserClient.java]: @erdemyerebasmaz  We need to remove the webln script from here.

**Review by roeierez — APPROVED** [review]

**Pull request merged**